doc/digital-receipt/implementation/digital-receipt-implementation.md

```diff
@@ -682,7 +682,7 @@ This chart shows the required data fields to visualize the hole dataset of the d
 | Amount  | 3.2  | mandatory  | yes  | Gross total price of service(s). The gross individual price, net total price, and net individual price, have to be calculated using the amount and either VAT rate or VAT amount  |
 | VATRate  | 20  | mandatory  | yes  | VAT rate as percentage  |
 | ftChargeItemCase  | 0x4154000000000003  | mandatory  | no  | Type of service or item according to the reference table in the appendix. It is used in order to determine the processing logic for the corresponding business transaction  |
-| ftChargeItemCaseData  | "{\"ftChargeItemCaseData\":[\"Mit Hafermilch\\r\\nund Zucker\"]}"  | optional  | currently not*  | Additional data about the service, currently accepted only in JSON format  |
+| ftChargeItemCaseData  |"ftChargeItemCaseData": "{\"ftChargeItemLines\":[\"Mit Hafermilch\\r\\nund Zucker\"]}"  | optional  | currently not*  | Additional data about the service, currently accepted only in JSON format  |
```

Comment:
Should be cbChargeItemLines IMO to be in sync with cbPayItemLines

Reply:
👌thx